src/main/java/com/shippo/model/CustomsDeclaration.java

```diff
@@ -248,12 +250,12 @@ public void setMetadata(Object metadata) {
 		this.metadata = metadata;
 	}
 
-	public Object getAdressImport() {
+	public Object getAddressImporter() {
 		return metadata;
```

Comment:
should this still be returning `metadata`?

Reply:
it should not.  Bad copypasta, thanks for catching.